src/chainparams.cpp

```diff
@@ -247,11 +245,10 @@ class CRegTestParams : public CChainParams {
     CRegTestParams() {
         strNetworkID = "regtest";
         consensus.nSubsidyHalvingInterval = 150;
-        consensus.nMajorityEnforceBlockUpgrade = 750;
-        consensus.nMajorityRejectBlockOutdated = 950;
-        consensus.nMajorityWindow = 1000;
-        consensus.BIP34Height = -1; // BIP34 has not necessarily activated on regtest
+        consensus.BIP34Height = 100000000; // BIP34 has not necessarily activated on regtest
```

Comment:
I just re added it with this check, tests with block v1 on regnet get rejected by the "obsolete version" check like  https://travis-ci.org/bitcoin/bitcoin/jobs/146661777 .  It makes sense as if BIP34Height is -1 then https://github.com/bitcoin/bitcoin/pull/8391/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR3515 will kick out.